design-proposals/isolate-emulator-thread-with-smt-fix/isolate-emulator-thread-with-smt-fix.md

```diff
@@ -54,13 +54,75 @@ Fix the actual scheduling of the virt-launcher to a node that will reject it.
 This bug addresses all KubeVirt users running VMs with isolateEmulatorThread on nodes with hyperthreading enabled - specifically DPDK and Realtime users.
 
 ## User Stories
-A user should be able to run VM with dedicatedCPUPlancement and isolateEmulatorThread enabled on a node with hyperthreading enabled, regardless of the amount of guest CPUs requested.
+* As a cluster admin, I want to set the cluster-wide IsolatedEmulatorThread behavior for Kubevirt by setting a knob on HCO.
+* as a VM owner, I should be able to run VM with dedicatedCPUPlancement and isolateEmulatorThread enabled on a node with hyperthreading enabled, regardless of the amount of guest CPUs requested.
 
 ## Repos
 - [kubevirt/kubevirt](https://github.com/kubevirt/kubevirt)
+- [kubevirt/hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator)
 
 # Bug fix design proposal
-The bug fix proposal introduces an alpha annotation `kubevirt.io/CPUManagerPolicyBetaOptions:full-pcpus-only`:
+The bug fix proposal introduces an alpha annotation to the kubevirt CR: `kubevirt.io/EmulatorThreadEvenParity`:
+```
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  annotations:
+    kubevirt.io/EmulatorThreadEvenParity:
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+      - EmulatorThreadEvenParity
+```
+This annotation will be supported only when the `EmulatorThreadEvenParity` featureGate is enabled.
+
+## Virt-API:
+When the following conditions are met:
+1. kubevirt CR has the annotation.
+2. kubevirt CR enabled the feature gate.
```

Comment:
```suggestion
2. The feature gate is enabled on the Kubevirt CR.
```

Please consider changing the order so the FG will be first, and the annotation on Kubevirt CR second.

Reply:
DONE